internal/pkg/service/appsproxy/appconfig/appconfig.go

```diff
@@ -19,15 +19,19 @@ import (
 // staleCacheFallbackDuration is the maximum duration for which the old configuration of an application is used if loading new configuration is not possible.
 const staleCacheFallbackDuration = time.Hour
 
+const notificationDelay = time.Second * 30
```

Comment:
`Delay`: it is not delay, the first notification is send immediately
- Please, add a comment.
- `notificationDelay` -> `notificationInterval` ?